### DISCUSSION TITLE
Feature request - synchronisation

### DISCUSSION THREAD

**Discussion by j-kn** [zidoro/pomatez#303] [Ideas]
First of all, thanks for creating this cool and helpful tool. 
It would be really great if I could synchronise pomatez (task list, settings, ...) running at my computer at work with the pomatez app that's running on my computer at home.
If this feature exists already I would be glad if someone could tell me where I can find it. If not I would kindly ask for this feature. 
Thanks!
J

**Comment by sekwah41** [comment]
It does not currently exist. Though if it were to we would need to look at either making an account system or possibly syncing the data via a cloud service such as Dropbox or google cloud.
